internal/vxproto/proto.go

```diff
@@ -854,12 +889,139 @@ func (vxp *vxProto) GetIMCModule(token string) IModuleSocket {
 	return nil
 }
 
+// GetIMCTopic is function which get topic info for imc topic token
+// The function may return nil if the topic token doesn't exist
+func (vxp *vxProto) GetIMCTopic(topic string) ITopicInfo {
+	if !vxp.HasIMCTopicFormat(topic) {
+		return nil
+	}
+
+	vxp.mutex.RLock()
+	defer vxp.mutex.RUnlock()
+
+	if info, ok := vxp.topics[topic]; ok && info != nil {
+		return info
+	}
+
+	return nil
```

Comment:
```suggestion
        return vxp.topics[topic]
```

Reply:
looks like well but it doesn't work due autocasting nil struct pointer to interface
proof: https://go.dev/play/p/P-cV8WBDlXc

> getKey1 false
> getKey2 false
> getKey1 false
> getKey2 true

Reply:
One more case when returning interface is not what it was actually invented for.